playground/model_trainer.py

```diff
@@ -111,6 +114,8 @@ def run_training(self, data, batch_size=32, epochs=1, save_interrupted=False):
         try:
             loss_history = self.fit_model(data, batch_size, epochs)
             endmodel_dir = os.path.join(self.experiment_dir, 'final')
```

Comment:
when plotting the history, please specify the `self.experiment_dir` where the plots should be saved. This code will put them in the root dir, next to `launcher.py` and this is not the right location. Edit it like that `plt.savefig(os.path.join(self.experiment_dir, 'loss_history.png'))`. It is also nice if the plot goes into a separate method, e.g. `def plot_loss_history(history)`. The plot code is likely to be expanded with more features like axis annotation, caption curve smoothing, legend and more and it will become cluttered if all is coded here.